src/languages/handlebars.js

```diff
@@ -5,32 +5,58 @@ Author: Robin Ward <robin.ward@gmail.com>
 Description: Matcher for Handlebars as well as EmberJS additions.
 Category: template
 */
-
 function(hljs) {
   var BUILT_INS = {'builtin-name': 'each in with if else unless bindattr action collection debugger log outlet template unbound view yield'};
+  function MUSTACHE_CONTENTS() {
+    return  {
+      className: 'name',
+      begin: /[a-zA-Z\.-]+/,
+      keywords: BUILT_INS,
+      starts: {
+        endsWithParent: true, relevance: 0,
+        contains: [
+          hljs.QUOTE_STRING_MODE
+        ]
+      }
+    };
+  }
+
   return {
     aliases: ['hbs', 'html.hbs', 'html.handlebars'],
     case_insensitive: true,
     subLanguage: 'xml',
     contains: [
-    hljs.COMMENT('{{!(--)?', '(--)?}}'),
+      hljs.COMMENT(/\{\{!--/, /--\}\}/),
+      hljs.COMMENT(/\{\{!/, /\}\}/),
+      // raw block {{{{raw}}}} verbatim xml {{{{/raw}} {{handlebars}}
+      {
+        className: 'template-tag',
+        begin: /\{\{\{\{\//, end: /\}\}\}\}/,
+        contains: [MUSTACHE_CONTENTS()]
```

Comment:
I think I'd prefer to see the open tag come first.  I was starting at this a while trying to figure out which item was doing what.  You can use a negative look-ahead to make sure there isn't a "/" immediately after so you don't flag the end tag by mistake.

Perhaps `// open/close ` comments might help also?

Our grammars don't tend to have  a ton of comments but I think they'd likely be more accessible if they did.